tools/github/scripts/Sync-AzsdkLabels.ps1

```diff
@@ -0,0 +1,82 @@
+[CmdletBinding(DefaultParameterSetName = 'RepositoryFile', SupportsShouldProcess = $true)]
+param (
+    [Parameter(Position = 0, Mandatory = $true)]
+    [ValidatePattern('^\w+/[\w-]+$')]
+    [string] $SourceRepository,
+
+    [Parameter(ParameterSetName = 'RepositoryFile')]
+    [ValidateScript({Test-Path $_ -PathType 'Leaf'})]
+    [string] $RepositoryFilePath = "$PSScriptRoot/../repositories.txt",
+
+    [Parameter(ParameterSetName = 'Repositories')]
+    [ValidateNotNullOrEmpty()]
+    [string[]] $Repositories = @(
+        'Azure/azure-sdk-for-cpp'
+        'Azure/azure-sdk-for-go'
+        'Azure/azure-sdk-for-java'
+        'Azure/azure-sdk-for-js'
+        'Azure/azure-sdk-for-net'
+        'Azure/azure-sdk-for-python'
+        'Azure/azure-sdk-tools'
+    ),
+
+    [Parameter(ParameterSetName = 'Languages')]
+    [ValidateNotNullOrEmpty()]
+    [string[]] $Languages = @('cpp', 'go', 'java', 'js', 'net', 'python'),
+
+    [Parameter()]
+    [switch] $Force
+)
+
+if (!(Get-Command -Type Application gh -ErrorAction Ignore)) {
+    throw 'You must first install the GitHub CLI: https://github.com/cli/cli/tree/trunk#installation'
+}
+
+if ($PSCmdlet.ParameterSetName -eq 'Languages') {
+    $Repositories = foreach ($lang in $Languages) {
+        "Azure/azure-sdk-for-$lang"
+    }
+}
+
+if ($PSCmdlet.ParameterSetName -eq 'RepositoryFile') {
+    $Repositories = Get-Content $RepositoryFilePath
+}
+
+# Filter out the source repository.
+$Repositories = $Repositories.Where({$_ -ne $SourceRepository})
+
+foreach ($repo in $Repositories) {
+    if ($Force -or $PSCmdlet.ShouldProcess(
+        "Cloning labels from $SourceRepository to $repo",
+        "Clone labels from $SourceRepository to $repo?",
+        "Clone labels")) {
+        $result = gh -R "$repo" label clone "$SourceRepository" --force 2>&1
+        if ($LASTEXITCODE) {
+            Write-Error "Failed to clone labels from $SourceRepository to ${repo}: $result"
+        }
+    }
+}
+
+<#
```

Comment:
@heaths shouldn't docs be first, by convention? I've never seen docs being at the bottom.

Reply:
I always put it at the bottom. It's not the important stuff when you want to read/edit the code. You should be looking at it with `help <cmdlet|script>` to better render it anyway.

Reply:
I would argue that natural language description in comments is easier to comprehend, and one should delve into the code only if they need deeper insight / more details. But to each their own, I guess.

Reply:
The `help` (`get-help`) cmdlet should be their first recourse for getting help. It's meant to be universal. If they have to open the script to get help, I argue the damage was already done. And if script writers are using non-doc-style comments that don't generate MAML (explicitly or implicitly), that's on them. ;)